_posts/2020-01-28-Tips-for-creating-open-source-projects-that-are-contributor-friendly.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Tips-for-creating-open-source-projects-that-are-contributor-friendly
+date: 2020-01-28 06:31:01
+categories: [open source]
```

Comment:
@bronzdoc This should be `open-source`. Check out other articles with this category for reference